### PR TITLE
add project Cover Letter Snippets to up-for-grabs projects.

### PR DESCRIPTION
**Pull request by mattcheah**

### CODE REVIEW

**Comment by ritwik12** [comment]
@mattcheah You have not mentioned any language in tags, is it correct?

**Comment by mattcheah** [comment]
Hi @ritwik12, my apologies, i forgot to do this. I have changed it now, thanks!

File: _data/projects/cover-letter-snippets.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- cover
+- letter
+- resume
+- job
+- description
+- parsing
+- database
+upforgrabs:
+  name: help wanted
+  link: https://github.com/mattcheah/cover-letter-snippets/issues
```
Comment (ritwik12) [inline _data/projects/cover-letter-snippets.yml]: @mattcheah Please mention the link to your label.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@mattcheah Thanks!!

**Pull request merged**